File: Basic concepts/5.function/Methodes_in_java/random/random.java
package random;
public class random {

    public static void main(String args[]){
        int max = 100;
        int min = 0;
        int i;
        for(i=0;i<100; i++){
             int  rand = (int)(Math.random()* (max - min + 1)) + min;
            System.out.print((rand)+" ");

        }
        
    }
    
}

File: Techniques/Random_Number_Generation/random.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {

     srand(time(0));

    for(int i=1; i<10; i++)
    {
         cout <<"random from 1 to 10 : "<<1+ (rand()%10)<<endl;
    }


    for (int x = 1; x <= 10; x++) {
        cout <<"Random from 1 to 6  :" <<1 + (rand() % 6) << endl;
    }

    return 0;
}

